service/relayer/migrations/20260626120000_relay_operation_key.down.sql

```
@@ -0,0 +1,33 @@
+
+ALTER TABLE "transaction"
+ALTER COLUMN transaction_hash SET NOT NULL;
+
+ALTER TABLE "transaction"
+ADD CONSTRAINT pk__transaction PRIMARY KEY (transaction_hash);
+
+ALTER TABLE "transaction"
+DROP COLUMN IF EXISTS operation_key;
+
+ALTER TABLE account
+ADD CONSTRAINT fk__account__transaction FOREIGN KEY (pending_transaction_hash) REFERENCES "transaction" (transaction_hash);
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)